examples/gallery/images/image.py

```diff
@@ -18,16 +17,15 @@
 fig = pygmt.Figure()
 fig.basemap(region=[0, 2, 0, 2], projection="X10c", frame=True)
 
-# place and center the GMT logo from the GMT website to the position 1/1
-# on a basemap, scaled up to be 3 cm wide and draw a rectangular border
-# around the image
+# Place and center ("+jCM") an image to the position ("+g") 1/1 on the current plot,
+# scale it to a width of 8 centimeters ("+w") and draw a rectangular border around it
 fig.image(
-    imagefile="https://www.generic-mapping-tools.org/_static/gmt-logo.png",
-    position="g1/1+w3c+jCM",
+    imagefile="https://www.emirates247.com/polopoly_fs/1.569712.1452551297!/image/image.jpg",
```

Comment:
Maybe still use the needle image but use a full URL:
```suggestion
    imagefile="https://oceania.generic-mapping-tools.org/cache/needle.jpg",
```